benchmarks/dynamo/common.py

```diff
@@ -1109,14 +1110,11 @@ def load(cls, model, example_inputs, eager_forward):
                 output_tensors.append(torch.empty_like(output))
 
             # The exact API is subject to change
-            exported = torch._export.export(model, example_inputs)
-            param_buffer_values = list(exported.state_dict.values())
-            flat_example_inputs = fx_pytree.tree_flatten_spec(
-                example_inputs, exported.call_spec.in_spec
+            exported = torch._export.export(
+                model, example_inputs, decomposition_table=select_decomp_table()
```

Comment:
Also we probably want to provide a specific API to target Export+AOTInductor. It's awkward for most end users to figure out which decomp table to pass in here.

Reply:
Ok for this PR. We can address it later.

Reply:
Makes sense -- do you have an opinion on where this API should live? In export or in inductor?

Reply:
Added the API to torch._export